circleguard/gui.py

```diff
@@ -230,13 +230,13 @@ def add_comparison_result(self, result):
                                                         r=result, r1=r1, r2=r2)
         result_widget = ComparisonResult(text, result, r1, r2)
         # set button signal connections (visualize and copy template to clipboard)
-        result_widget.button.clicked.connect(partial(self.main_window.main_tab.visualize, [result_widget.replay1, result_widget.replay2]))
+        result_widget.button.clicked.connect(partial(self.main_window.main_tab.visualize, [result_widget.replay1, result_widget.replay2], result_widget.replay1.map_id))
         result_widget.button_clipboard.clicked.connect(partial(self.copy_to_clipboard,
                 get_setting("template_replay_steal").format(r=result_widget.result, r1=result_widget.replay1, r2=result_widget.replay2)))
         # remove info text if shown
         if not self.main_window.results_tab.results.info_label.isHidden():
             self.main_window.results_tab.results.info_label.hide()
-        self.main_window.results_tab.results.layout.addWidget(result_widget)
+        self.main_window.results_tab.results.layout.insertWidget(0,result_widget)
```

Comment:
does this change actually *do* anything? I understand it will make the info label always be at the end of the layout, but that shouldn't matter - it's hidden.

Reply:
addWidget puts the new widget at the bottom of the layout, insertWidget with an index of 0 adds it at the top of the layout

Reply:
Ah I see, that should have been obvious.

I'm not sure this behavior is as intuitive as the previous behavior of "newest at the bottom", though it does make more sense. We should update the initial info label to mention it puts the newest results at the top.